### PR TITLE
Update API version.

### PR DESCRIPTION
**Pull request by kegor**
Replicated changes from https://github.com/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pull/55.

### CODE REVIEW

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 27
           

Complexity increasing per file
==============================
- src/Products/Annotation/Annotator/BaseAnnotator.cs  1
- src/Products/Annotation/Util/DocumentTypesConverter.cs  1
- src/Products/Annotation/Util/AnnotationMapper.cs  16
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  2
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  3
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  2
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  1
         

Clones added
============
- src/Products/Annotation/Annotator/TextRedactionAnnotator.cs  2
- src/Products/Annotation/Annotator/TexUnderlineAnnotator.cs  1
- src/Products/Annotation/Annotator/TextReplacementAnnotator.cs  1
- src/Products/Annotation/Annotator/TexStrikeoutAnnotator.cs  4
- src/Products/Annotation/Annotator/PolylineAnnotator.cs  1
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  6
- src/Products/Annotation/Annotator/ResourceRedactionAnnotator.cs  5
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  2
- src/Products/Annotation/Annotator/DistanceAnnotator.cs  3
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  4
- src/Products/Annotation/Annotator/PointAnnotator.cs  2
- src/Products/Annotation/Annotator/AreaAnnotator.cs  2
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  2
         

Clones removed
==============
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483022&bid=17967732)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 15
           

Complexity increasing per file
==============================
- src/Products/Annotation/Util/DocumentTypesConverter.cs  1
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  1
- src/Products/Annotation/Annotator/BaseAnnotator.cs  1
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  3
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  2
- src/Products/Annotation/Util/AnnotationMapper.cs  16
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  2
         

Clones added
============
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  2
- src/Products/Annotation/Annotator/DistanceAnnotator.cs  3
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  2
- src/Products/Annotation/Annotator/PointAnnotator.cs  2
- src/Products/Annotation/Annotator/ResourceRedactionAnnotator.cs  5
- src/Products/Annotation/Annotator/TexStrikeoutAnnotator.cs  4
- src/Products/Annotation/Annotator/TexUnderlineAnnotator.cs  1
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  4
- src/Products/Annotation/Annotator/PolylineAnnotator.cs  1
- src/Products/Annotation/Annotator/TextReplacementAnnotator.cs  1
- src/Products/Annotation/Annotator/AreaAnnotator.cs  2
- src/Products/Annotation/Annotator/TextRedactionAnnotator.cs  2
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  6
         

Clones removed
==============
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483021&bid=17967731)

File: src/Products/Annotation/Controllers/AnnotationApiController.cs [hunk 1]
```diff
@@ -30,365 +27,273 @@ namespace GroupDocs.Annotation.WebForms.Products.Annotation.Controllers
...
-            // create annotation application configuration
-            AnnotationConfig config = new AnnotationConfig
-            {
-                // set storage path
-                StoragePath = directoryUtils.FilesDirectory.GetPath(),
-            };
-
-            // initialize Annotation instance for the Image mode
-            annotationImageHandler = new AnnotationImageHandler(config);
+            GlobalConfiguration = new Common.Config.GlobalConfiguration();
```
Comment (egorovpavel) [inline src/Products/Annotation/Controllers/AnnotationApiController.cs:38]: Codacy found an issue: [Remove this assignment of 'GlobalConfiguration' or initialize it statically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483021)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Annotation/Util/AnnotationMapper.cs  16
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  2
- src/Products/Annotation/Annotator/BaseAnnotator.cs  1
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  2
- src/Products/Annotation/Util/DocumentTypesConverter.cs  1
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  3
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  1
         

Complexity decreasing per file
==============================
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -1
         

Clones added
============
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  6
- src/Products/Annotation/Annotator/PolylineAnnotator.cs  1
- src/Products/Annotation/Annotator/TextReplacementAnnotator.cs  1
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  4
- src/Products/Annotation/Annotator/TexStrikeoutAnnotator.cs  4
- src/Products/Annotation/Annotator/AreaAnnotator.cs  2
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  2
- src/Products/Annotation/Annotator/PointAnnotator.cs  2
- src/Products/Annotation/Annotator/TexUnderlineAnnotator.cs  1
- src/Products/Annotation/Annotator/TextRedactionAnnotator.cs  2
- src/Products/Annotation/Annotator/ResourceRedactionAnnotator.cs  5
- src/Products/Annotation/Annotator/DistanceAnnotator.cs  3
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  2
         

Clones removed
==============
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483021&bid=17967731)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 16
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  2
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  1
- src/Products/Annotation/Annotator/BaseAnnotator.cs  1
- src/Products/Annotation/Util/AnnotationMapper.cs  16
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  3
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  2
- src/Products/Annotation/Util/DocumentTypesConverter.cs  1
         

Complexity decreasing per file
==============================
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -1
         

Clones added
============
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  4
- src/Products/Annotation/Annotator/AreaAnnotator.cs  2
- src/Products/Annotation/Annotator/PointAnnotator.cs  2
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  2
- src/Products/Annotation/Annotator/TextRedactionAnnotator.cs  2
- src/Products/Annotation/Annotator/ResourceRedactionAnnotator.cs  5
- src/Products/Annotation/Annotator/DistanceAnnotator.cs  3
- src/Products/Annotation/Annotator/TextReplacementAnnotator.cs  1
- src/Products/Annotation/Annotator/TexStrikeoutAnnotator.cs  4
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  2
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  6
- src/Products/Annotation/Annotator/TexUnderlineAnnotator.cs  1
- src/Products/Annotation/Annotator/PolylineAnnotator.cs  1
         

Clones removed
==============
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483022&bid=17967732)

File: src/Products/Annotation/Controllers/AnnotationApiController.cs (hunk 1 above)
Comment (bobkovalex) [inline src/Products/Annotation/Controllers/AnnotationApiController.cs:38]: Codacy found an issue: [Remove this assignment of 'GlobalConfiguration' or initialize it statically.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483022)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 16
           

Complexity increasing per file
==============================
- src/Products/Annotation/Util/AnnotationMapper.cs  16
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  3
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  2
- src/Products/Annotation/Annotator/BaseAnnotator.cs  1
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  2
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  1
- src/Products/Annotation/Util/DocumentTypesConverter.cs  1
         

Complexity decreasing per file
==============================
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -1
         

Clones added
============
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  2
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  6
- src/Products/Annotation/Annotator/DistanceAnnotator.cs  3
- src/Products/Annotation/Annotator/TextRedactionAnnotator.cs  2
- src/Products/Annotation/Annotator/PointAnnotator.cs  2
- src/Products/Annotation/Annotator/ResourceRedactionAnnotator.cs  5
- src/Products/Annotation/Annotator/PolylineAnnotator.cs  1
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  4
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  2
- src/Products/Annotation/Annotator/TexStrikeoutAnnotator.cs  4
- src/Products/Annotation/Annotator/TextReplacementAnnotator.cs  1
- src/Products/Annotation/Annotator/TexUnderlineAnnotator.cs  1
- src/Products/Annotation/Annotator/AreaAnnotator.cs  2
         

Clones removed
==============
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483022&bid=17967732)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
           

Complexity increasing per file
==============================
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  3
- src/Products/Annotation/Annotator/BaseAnnotator.cs  1
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  2
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  1
- src/Products/Annotation/Util/AnnotationMapper.cs  16
- src/Products/Annotation/Util/DocumentTypesConverter.cs  1
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  2
         

Complexity decreasing per file
==============================
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -1
         

Clones added
============
- src/Products/Annotation/Annotator/ArrowAnnotator.cs  2
- src/Products/Annotation/Annotator/ResourceRedactionAnnotator.cs  5
- src/Products/Annotation/Annotator/PointAnnotator.cs  2
- src/Products/Annotation/Annotator/TextRedactionAnnotator.cs  2
- src/Products/Annotation/Annotator/TexUnderlineAnnotator.cs  1
- src/Products/Annotation/Annotator/WatermarkAnnotator.cs  4
- src/Products/Annotation/Annotator/PolylineAnnotator.cs  1
- src/Products/Annotation/Annotator/TextHighlightAnnotator.cs  2
- src/Products/Annotation/Annotator/TextReplacementAnnotator.cs  1
- src/Products/Annotation/Annotator/AreaAnnotator.cs  2
- src/Products/Annotation/Annotator/DistanceAnnotator.cs  3
- src/Products/Annotation/Annotator/TexStrikeoutAnnotator.cs  4
- src/Products/Annotation/Annotator/TextFieldAnnotator.cs  6
         

Clones removed
==============
+ src/Products/Annotation/Controllers/AnnotationApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483021&bid=17967731)

**Review by Pavel-Teplitsky — APPROVED** [review]

**Pull request merged**